packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,139 @@
+
+Event hooks are outbound calls from Okta, sent when specified events occur in your org. They take the form of HTTPS REST calls to a URL you specify, encapsulating information about the events in JSON objects in the request body. These calls from Okta are meant to be used as triggers for process flows within your own software systems.
+
+To handle event hook calls from Okta, you need to implement a web service with an Internet-accessible endpoint. It's your responsibility to develop the code and to arrange its hosting on a system external to Okta. Okta defines the REST API contract for the requests that it will send.
+
+Event hooks are related to, but different from, Okta [inline hooks](/use_cases/inline_hooks/). Event hooks are meant to provide information about events that occurred, not offer a way to affect the underlying Okta process flow. For that, you need to use inline hooks. Also, unlike inline hooks, event hooks are asynchronous calls, meaning that the process flow that triggered the event hook continues without stopping or waiting for any response from your external service.
+
+## Which Events are Eligible?
+
+During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures. To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `event-hook-eligible`:
+
+[https://developer.okta.com/docs/api/resources/event-types/?q=event-hook-eligible](/docs/api/resources/event-types/?q=event-hook-eligible)
+
+## Requests Sent by Okta
+
+The requests sent from Okta to your external service are HTTPS requests. POST requests are used for the ongoing delivery of events, and a one-time GET request is used for verifying your endpoint.
+
+### One-Time Verification Request
+
+After registering an event hook, but before you can use it, you need to have Okta make a one-time GET verification request to your endpoint, passing your service a verification value that it needs to send back. This serves as a test confirming that you control the endpoint.
+
+This one-time verification request is the only GET request Okta will send to your external service, while the ongoing requests to notify your service of event occurrences will be HTTPS POST requests. Your web service can use the GET versus POST distinction to implement logic to handle this special one-time request.
+
+The way your service needs to handle this one-time verification is as follows: The request from Okta will contain an HTTP header named `X-Okta-Verification-Challenge`. Your service needs to read the value of that header and return it in the response body, in a JSON object named `verification`, i.e.: `{ "verification" : "value_from_header" }`. Note that the value comes to you in an HTTP header, but you need to send it back in a JSON object.
```

Comment:
Under the sub-section "One-Time Verification Request", the last two paragraphs contain implementation related info. IMHO, it might be better to move these two paragraphs in under "Verifying an Event Hook" (in the section _Event Hooks Set Up_) because that is where we talk about how to make a POST request to the /api/v1/eventHooks/${eventHookId}/verify API.

We can still keep the first paragraph in the sub-section "One-Time Verification Request" where we describe why we do this one-time verification. At the end of this paragraph, we can simply add a link to the upcoming section "Verifying an Event Hook" on how to implement verification.

Reply:
@ankurdatta-okta, customers have to code logic into their service to handle this one-time verification request, because the same service that handles ongoing requests will have to be able to handle this one too.

The "Verifying an Event Hook" procedure, on the other hand, is manually performed by a human being.

Isn't it clearer to keep those two things separate: Logic you need to implement in your code as you develop your software, versus setup steps you need to perform manually?

Reply:
Sounds good. I think I understood now why you kept the two things separate. It would be nice if we add a hyperlink to the section on "Verifying an Event Hook" towards the end of the section on "One-Time Verification Request". For someone unfamiliar with the verification flow, this will help them relate the two separate steps.

Reply:
Added a link from "One-Time Verification Request" section to "Verifying an Event Hook" section (all changes are in new pull request: https://github.com/okta/okta-developer-docs/pull/227)